src/routes/protected_areas.js

```diff
@@ -352,6 +352,36 @@ module.exports = (errorHandler, paService) => {
       })
   )));
 
+  /**
+   * @apiGroup s_hf
+   * @api {get} /pa/:category/:se_type/hf/timeline SETimeLineInPA
+   * @apiName SETimeLineInPA
```

Comment:
You didin't change the name `HFTimeLineInPA` as in the other areas

Reply:
Done